pkg/gitutils/manifests.go

```diff
@@ -167,10 +104,24 @@ func CopyPatchManifests(wt *gogit.Worktree, filePath string, clusterPath string,
 	if err != nil {
 		return fmt.Errorf("failed to create kustomization.yaml: %s", err)
 	}
+	defer file.Close()
```

Comment:
General comment:
Can you change the defers to: `defer _ = xxx.Close()` or else my IDE will complain about the return value not being used. Not sure what IDE you use.